src/components/AddProject.vue

```diff
@@ -0,0 +1,116 @@
+<template>
+<div id="add-project-container">
+    <div class="field">
+        <label for="project-title">TITLE</label>
+        <input type="text" id="project-title" v-model="title">
+    </div>
+    <div class="field">
+        <label for="project-details">DETAILS</label>
+        <textarea name="" id="project-details" cols="30" rows="10" v-model="detail"></textarea>
+    </div>
+    <button @click="addProject" class="btn" v-if="!this.id">Add Project</button>
+    <button @click="updateProject" class="btn" v-if="this.id">Update Project</button>
+</div>
+</template>
+
+<script>
+export default {
+    name: 'AddProject',
+    props: {
+        id: String
+    },
+    data() {
+        return {
+            title: "",
+            detail: "",
+            project: [],
+            projectTemp: [],
+        };
+    },
+
+    methods: {
+        addProject() {
+            this.project.push({
+                'title': this.title,
+                'detail': this.detail,
+                'completed': false,
+                'ongoing': true
```

Comment:
@anubha-innotech `ongoing` is an extra variable, we should remove it, ongoing means project is not completed yet (false).